static/css/style.css

```diff
@@ -849,18 +851,27 @@ fieldset[disabled] .source-buttons .btn-primary.active {
   border: 1px solid transparent;
   white-space: nowrap;
   line-height: 1.42857143;
-  border-radius: 0px;
   -webkit-user-select: none;
   -moz-user-select: none;
   -ms-user-select: none;
   user-select: none;
   text-transform: uppercase;
   text-shadow: none;
-  letter-spacing: .15em;
+  letter-spacing: .05em;
   font-weight: bold;
-  font-size: .9em;
```

Comment:
Yeah, the layout of the docs page still isn't the same.  I think you need to only apply these changes to where you're using them, which I think is only the "Return" button on the post page.